src/Common/src/Interop/Unix/libssl/SslConnectionInfo.cs

```diff
@@ -26,7 +26,8 @@ internal SslConnectionInfo(SafeSslHandle sslContext)
                 out DataCipherAlg,
                 out KeyExchangeAlg,
                 out DataHashAlg,
-                out DataKeySize))
+                out DataKeySize,
+                out DataHashKeySize))
             {
                 throw Interop.OpenSsl.CreateSslException(SR.net_ssl_get_connection_info_failed);
             }
```

Comment:
I think this would be good to note in the code. Also opening an issue for this and using that number in the TODO comment is a good idea since #3362 is closed and #4128 is going to come through and just delete these TODO comments.